driver-core/src/test/unit/com/mongodb/UriOptionsTest.java

```diff
@@ -47,9 +47,6 @@ public void shouldPassAllOutcomes() {
         // No CANONICALIZE_HOST_NAME support https://jira.mongodb.org/browse/JAVA-4278
         assumeFalse(getDescription().equals("Valid auth options are parsed correctly (GSSAPI)"));
 
-        // https://jira.mongodb.org/browse/JAVA-5834
-        assumeFalse(getFilename().equals("proxy-options.json"));
```

Comment:
`proxy-options.json` was not added to our codebase by https://github.com/mongodb/mongo-java-driver/pull/1180 (in fact, no spec tests were added), despite them having been added in https://jira.mongodb.org/browse/DRIVERS-1357. I wonder if @vbabanin remembers the reason, and if it was done on purpose.

Reply:
Got a response - it wasn't the case that we omitted them intentionally, because they did not fit our behavior.